collections/_posts/2024-03-02-gsoc.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: Typelevel Summer of Code 2024
+category: technical
+
+meta:
+  nav: blog
+  author: armanbilge
+---
+
+We are excited to share that Typelevel will be participating in [Google Summer of Code 2024][GSoC], thanks to the gracious support of the [Scala Center]! If you are a student, this is a wonderful opportunity to spend your summer working on Scala open source projects with mentorship from Typelevel maintainers, while earning a stipend.
+
+This year we have [several project ideas and mentors lined up][ideas] spanning AI/ML, serverless, data streaming, observability, systems programming, and more. We will continue adding ideas, so please keep checking back! Also, if you have an idea of your own, we would love to hear it: Typelevel members have a wide range of interests and a special fondness for building “at the cutting edge of some incredibly bizarre stuff.”
+
+We look forward to welcoming you to the Typelevel community and we hope that this program will be only the beginning of your open source journey. [Applications open on March 18!][apply] Until then, please join us on [Discord] or email us at [gsoc@typelevel.org]; we look forward to getting to know you :)
+
+## 2023 recap
+
+We are proud of our GSoC contributors' accomplishments last year:
+
+* [Hieu] implemented a pure FP Scala [WebSocket client] in [http4s Ember] that supports JVM, Node.js, and Scala Native. His work is being adopted across the ecosystem, such as by the [Kubernetes client].
+* [Sherrie] carefully reviewed [otel4s] and [designed new APIs] that better follow the [Open Telemetry] specification. Her enhancements shipped as part of the [v0.3.0 release].
+* [Antonio] demonstrated the awesome potential of [io_uring] when [http4s Ember] performed [more than 3x faster in benchmarks] thanks to his [I/O integrated runtime]. He is returning this year as a mentor on multiple follow-up projects.
+
+To learn more about their projects, check out the [lightning talks] they gave at the [2023 Northeast Scala Symposium]. Their success would not have been possible without the support and mentorship of our community, especially Diego Alonso, Ross Baker, Chris Davenport, Brian Holt, Maksym Ochenashko, and Daniel Spiewak. Thank you for your enthusiasm and generosity; it made an impression on our students, and it made an impression on me :)
+
+Lastly, we were surprised by the breadth of the response we got to our announcement last year: there are many people eager to contribute. Even if you are not eligible to participate in GSoC, you are *always* welcome to join the Typelevel community and contribute to our projects!
+
+[Scala Center]: https://scala.epfl.ch/
+[GSoC]: https://summerofcode.withgoogle.com/
+[ideas]: /gsoc/ideas
```

Comment:
I couldn't figure out how to make this URL work in the local preview, since it hosts the site at a sub-path :(